pkg/kubelet/kuberuntime/kuberuntime_container_test.go

```diff
@@ -79,9 +80,14 @@ func TestRemoveContainer(t *testing.T) {
 
 	err = m.removeContainer(containerID)
 	assert.NoError(t, err)
-	// Verify container log is removed
 
-	assert.Equal(t, []string{expectedContainerLogPath, expectedContainerLogPathRotated, expectedContainerLogSymlink}, fakeOS.Removes)
+	actualRemoves := make([]string, 0, len(fakeOS.Removes))
```

Comment:
looks like we could simply sort `fakeOS.Removes` in place.

Reply:
@knight42 assert.ElementsMatch?

Reply:
Thanks for pointing it out! Fixed.